src/storage/storage.py

```diff
@@ -1,21 +1,21 @@
 """
-src/storage/storage.py
-======================
+Storing inventory items
 """
 import collections
 
-InventoryItem = collections.namedtuple('InventoryItem', 'type stream payload expires tag')
+InventoryItem = collections.namedtuple(
+    'InventoryItem', 'type stream payload expires tag')
 
 
-class Storage(object):
-    """Base class for storing inventory (extendable for other items to store)"""
+class Storage(object):  # pylint: disable=too-few-public-methods
+    """Base class for storing inventory(extendable for other items to store)"""
```

Comment:
Why are you removing the space?

Reply:
By mistake. Trying to make the line within 79 length. Forget to create new line. But Fixed now. Thanks!

Reply:
I see. Such economy makes sense only in tests where docstrings are showed below the test method names.